__tests__/AutocompleteInput.spec.js

```diff
@@ -72,4 +72,50 @@ describe('<AutocompleteInput />', () => {
 
     expect(textInput.props).toEqual(expect.objectContaining(props));
   });
+
+  it('should render default <FlatList /> if no custom one is supplied', () => {
+    const testRenderer = renderer.create(<Autocomplete data={ITEMS} />);
+    const autocomplete = testRenderer.root;
+    const list = autocomplete.findByType(FlatList);
+
+    expect(list.props.data).toEqual(ITEMS);
+  });
+
+  it('should only pass props in flatListProps to <FlatList />', () => {
+    // Using keyExtractor isn't important for the test, but prevents a warning
+    const keyExtractor = (_, index) => `key-${index}`;
+    const flatListProps = { foo: 'bar', keyExtractor };
+    const otherProps = { baz: 'qux' };
+    const testRenderer = renderer.create(
+      <Autocomplete data={ITEMS} flatListProps={flatListProps} {...otherProps} />
+    );
+    const autocomplete = testRenderer.root;
+    const list = autocomplete.findByType(FlatList);
+
+    expect(list.props).toEqual(expect.objectContaining(flatListProps));
+    expect(list.props).toEqual(expect.not.objectContaining(otherProps));
+  });
+
+  it('should render a custom result list', () => {
+    const testRenderer = renderer.create(
+      <Autocomplete
+        data={ITEMS}
+        renderResultList={({ data, style }) => (
+          <View style={style}>
+            {data.map((item, index) => (
+              // eslint-disable-next-line react/no-array-index-key
```

Comment:
Hope you're ok with this. I figured it was fine to disable linting for a test. Adding real IDs would have complicated the test.